include/NovelRT/Experimental/Windowing/Glfw/Windowing.Glfw.h

```diff
@@ -0,0 +1,28 @@
+// Copyright © Matt Jones and Contributors. Licensed under the MIT Licence (MIT). See LICENCE.md in the repository root
+// for more information.
+
+#ifndef NOVELRT_EXPERIMENTAL_WINDOWING_GLFW_H
+#define NOVELRT_EXPERIMENTAL_WINDOWING_GLFW_H
+
+/**
+ * @Brief The default GLFW3 implementation of the Windowing plugin API.
+ */
+namespace NovelRT::Experimental::Windowing::Glfw
+{
+    class GlfwWindowingDevice;
+}
+
+// Windowing.Glfw dependencies
+#include "../../../Maths/GeoVector2F.h"
+#include "../Windowing.h"
+#include <cstdint>
+
+// TODO: Figure out if this is required
+#define GLFW_INCLUDE_VULKAN
```

Comment:
Is it?

Reply:
pretty sure it is if we want the Vulkan stuff from GLFW but I can look into it again.

Reply:
Looks like this should only be needed if you're not already including the vulkan header prior to including GLFW. If you're specifying it directly (the vulkan header) before this include, feel free to remove this def and it _should_ work fine.

Reply:
I'm not - because GLFW has no context of the Vulkan plugin.

I think we're better safe than sorry here.